app/lib/zeebe-api/zeebe-api.js

```diff
@@ -362,14 +362,10 @@ class ZeebeAPI {
       parsed = new X509Certificate(cert);
     } catch (err) {
       this._log.error('Failed to parse custom SSL certificate:', err);
-
-      return;
     }
 
-    if (parsed.issuer !== parsed.subject) {
-      this._log.error('Custom SSL certificate is not a root certificate');
-
-      return;
+    if (parsed && parsed.issuer !== parsed.subject) {
+      this._log.error('Custom SSL certificate appears to be not a root certificate');
```

Comment:
We could make this a warning. Given the new scope that sounds more reasonable to me.

Reply:
OK